BREAKING.md

```diff
@@ -50,6 +51,9 @@ sometimes followed by a colon and an inner error message when applicable.
 Users will not be allowed to do operations on a deleted directory. Users can subscribe to `disposed` event to know if a sub directory is deleted. Accessing deleted sub directory
 will throw `UsageError` exception now.
 
+### Removing Commit from TreeEntry and commits from SnapShotTree
+TreeEntry will no longer contain the property Commit and the same goes for commits from ISnapshotTree.
```

Comment:
If there's a replacement for this or any other information which may be useful for clients migrating off using this, please add it here.

Reply:
I don't think there is a replacement - they are not used at all, I believe.